src/main/kotlin/no/nav/familie/klage/behandling/BehandlingRepository.kt

```diff
@@ -44,4 +46,27 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
             WHERE b.ekstern_behandling_id = :eksternBehandlingId AND f.fagsystem = :fagsystem"""
     )
     fun findByEksternBehandlingIdAndFagsystem(eksternBehandlingId: String, fagsystem: String): Behandling
+
+    @Query(
+        """
+            SELECT 
+             b.id,
+             b.fagsak_id,
+             f.fagsak_person_id,
+             b.status,
+             b.opprettet_tid opprettet,
+             b.klage_mottatt mottatt_dato,
+             b.resultat,
+             v.arsak,
+             b.vedtak_dato vedtaksdato
+            FROM behandling b
+            JOIN fagsak f ON f.id = b.fagsak_id
+            LEFT JOIN vurdering v ON v.behandling_id = b.id
+            WHERE f.ekstern_id = :eksternId AND f.fagsystem = :fagsystem
+        """
+    )
+    fun finnBehandlinger(
+        @Param("eksternId") eksternId: String,
```

Comment:
`eksternFagsakId`

Reply:
Har vi vurdert gevinst vs kost ved å ha denne spesialspørringen kontra tre repository-kall som mappes om til KlageBehandlingDto direkte?

Reply:
Skjønner at spesialspørringer er nyttig når volumet blir stort, men vet ikke helt om vi er der?

Reply:
Fikset (og snakket om spørringen)

Reply:
Fortsetter med spesialspørring siden det fort blir en eller to joins til etter neste uke